packages/store/addon/-private/system/deprecated-evented.js

```diff
@@ -27,10 +27,15 @@ if (DEBUG) {
     return deprecations;
   };
 
+  /**
+   * `DeprecatedEvented` is a mixin that proxies to the `Ember.Evented`
+   * mixin without logging deprecation warnings.
```

Comment:
```suggestion
   * mixin while logging deprecations. It is used by classes that were previously instrumented with
   * Evented whose evented APIs are now deprecated.
```

Overall LGTM but added some clarification. The `_has` being a workaround to not log the deprecation exists because we don't want to flood the user with deprecation notices while we are merely checking for whether they've registered a deprecated event which we will need to fire.